### ISSUE TITLE
Add option to print command to standard output

### ISSUE THREAD

**Issue by telotortium** [simonw/llm-cmd#14]
I use `llm cmd` because it takes care of the prompting to extract just a command from the LLM. Sometimes I don't want to edit and run the command, but just copy it. I can always just manually copy and paste, but it might be nice to use this from a keyboard shortcut or something like that, in which case the readline interface gets in the way.

Neither `llm cmd ... </dev/null` nor `llm cmd ... | cat` works as a workaround right now.

**Comment by dbosk** [comment]
I don't think that an option is necessary. Just to add the following in the right place:
```python
if os.isatty(sys.stdin.fileno()):
    # do what we do now
else:
    # just print plain text to stdout
```
